javascript/javascript_in_the_real_world/linting.md

```diff
@@ -62,22 +70,25 @@ For most people using the default ESLint ruleset, there will be no special setup
 
 Some community plugins, such as [eslint-config-airbnb-base](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base), turn on some stylistic rules that may clash with what Prettier formats. If you wish to use a plugin like `eslint-config-airbnb-base` and Prettier together, you will also need to install [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) which will turn off any of the ESLint rules that clash with Prettier. If you are using the default ESLint ruleset, you will not need this.
 
-### Template repositories
+### Assignment
 
-With the last few projects, you might have felt that setting up Webpack involved a fair few files and configuration, and that you may have had to look at what you configured before to copy and paste the configuration you want to reuse. You may also have noticed that whenever you create a new repository on Github, there is an option near the top for a `Repository template`.
+<div class="lesson-content__panel" markdown="1">
 
-This is where template repositories can come very much in handy. Any of your existing repositories can be converted to a template in its settings (right under where you can rename the repository, there is a checkbox for whether the repository is a template or not). If you check this box, congratulations, that's all you need to do! Now when you go to create a new repository, the `Repository template` dropdown will have any templates listed for you to select. Selecting one will mean your new repository will be a copy of the chosen template, not an empty one!
+1. Read this article that goes more into [the value of linters and how they work](https://hackernoon.com/how-linting-and-eslint-improve-code-quality-fa83d2469efe).
+1. Watch this [short intro to Prettier](https://www.youtube.com/watch?v=hkfBvpEfWdA) by its creator.
+1. Go to [Prettier's online playground](https://prettier.io/playground) and give it a test drive. Go ahead and copy/paste some of your old JavaScript code into that editor and see what happens.
 
-If you find yourself reusing a lot of setup code for multiple projects, you can make a new repository with all of the setup code you need then mark it as a template. Now you can select that template when creating a new project repository to save time getting set up, letting you dive into working on the project itself sooner!
```

Comment:
Anyone can provide their thoughts on any PR, so long as they're genuine and sincere comments of course. At "worst", you'll find out there are things you perhaps didn't consider or realise, and you get stuff out of the discussion.

About your comments, that was considered and whenever there are changes (especially significant ones), you're always going to get people missing stuff (e.g. people ahead who won't have spotted or want to redo past lessons just because of some changes). That being said, I do like your suggestion a lot and didn't consider that. Something like "Remember template repositories? etc..." and a link back to Revisiting Webpack, that'll be a reasonable solution :muscle: